tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+	protected function construct() {
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Events__Main', '4.4' ); // due to using tribe() from Tribe-Common as of 4.4
+		$this->add_required_plugin( 'Tribe__Events__Tickets__Eventbrite__Main', '4.4' ); // just to match version of Tribe__Events__Main
+
+		// Set the extension's TEC URL
+		$this->set_url( 'https://theeventscalendar.com/extensions/eventbrite-additional-options/' );
+	}
+
+	/**
+	 * Adds settings options
+	 */
+	public function add_settings() {
+		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		$fields = array(
+			$this->opts_prefix . 'heading' => array(
+				'type' => 'html',
+				'html' => '<h3>' . esc_html__( 'Eventbrite Additional Options', 'tribe-extension' ) . '</h3>',
+			),
+			$this->opts_prefix . 'helper_text' => array(
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'The following fields are from an Extension. These additional Eventbrite options allow you to change how the Eventbrite Tickets add-on operates, such as moving the ticket location on Single Event view, displaying tickets for Private Eventbrite events, and more.', 'tribe-extension' ) . '</p>',
+			),
+
+			$this->opts_prefix . 'content_before' => array(
+				'type'				=> 'text',
+				'label'				=> esc_html__( '<h3> text to display Before/Above iframe', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default there is no content before the iframe. You might want to try something like "Eventbrite Tickets". It will be output as an <h3> heading with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
+				'validation_type'	=> 'textarea', // runs wp_kses()
+			),
+
+			$this->opts_prefix . 'content_after' => array(
+				'type'				=> 'text',
+				'label'				=> esc_html__( '<span> text to display After/Below iframe', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default there is no content after the iframe. It will be output as a <span> with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
+				'validation_type'	=> 'textarea', // runs wp_kses()
+			),
+
+			// see https://central.tri.be/issues/62402
+			$this->opts_prefix . 'iframe_px' => array(
+				'type'				=> 'text',
+				'label'				=> esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
+				'default'			=> 200, // required else positive_int validation will throw an error each time settings are saved if this field is left blank
+				'tooltip'			=> esc_html__( 'The default is 200.', 'tribe-extension' ),
+				'validation_type'	=> 'positive_int',
+			),
+
+			$this->opts_prefix . 'locale' => array(
+				'type'				=> 'dropdown',
+				'label'				=> esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
+				'default'			=> 'en_US',
+				'tooltip'			=> sprintf( esc_html__( 'This affects the API call. The default is %sUnited States (en_US)%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
```

Comment:
Two comments before, you said, "Defaults are the norm and should generally be set," so do you mean _mention_ of the default in the Tooltip should be removed? Or do you mean setting the `'default'` value should actually be removed?